src/components/panes/Quality/QualityModal/QualityModal.tsx

```diff
@@ -94,7 +95,13 @@ export function QualityModal({ isModalOpen, setIsModalOpen, latency, jitter, pac
               <TableRow>
                 <TableCell>
                   <div className={classes.iconContainer}>
-                    <Tooltip title={toolTipContent.roundTripTime} interactive leaveDelay={250}>
+                    <Tooltip
+                      title={toolTipContent.roundTripTime}
+                      interactive
+                      leaveDelay={250}
+                      leaveTouchDelay={3500}
+                      enterTouchDelay={100}
```

Comment:
The delay of 100 feels a little funny on my Android. Would there be any problems if we made it 0?

Also, I think the leaveTouchDelay should be increased. I couldn't finish reading some of these in 3.5 seconds. Since these tooltips close when the user taps away from them, I think the leave delay could be much higher. Maybe even 15 seconds?